superset-frontend/src/SqlLab/main.less

```diff
@@ -300,7 +300,7 @@ div.Workspace {
   .schemaPane-enter-done,
   .schemaPane-exit {
     transform: translateX(0);
-    z-index: 1020;
+    z-index: 7;
```

Comment:
I think we have some z-index variables. It would be nice to manage them all in one place:

https://github.com/apache/superset/blob/75f927b9ebe4174d9af37ac4d4ce1a043cee77f1/superset-frontend/stylesheets/less/variables.less#L128

Reply:
I agree with Jesse, but let not block for this enhancement